### PR TITLE
Denoise bilateral filter

### PR DESCRIPTION
**Pull request by PaoloAst**
Allowing bilateral filter to be moved in the correct group

### CODE REVIEW

**Comment by TurboGit** [comment]
Good catch! Thanks.

File: src/iop/bilateral.cc [hunk 1]
```diff
@@ -69,9 +69,9 @@ const char *name()
   return _("denoise (bilateral filter)");
 }
 
-int groups()
+int groups() 
 {
-  return IOP_GROUP_CORRECT;
+return dt_iop_get_group("input color profile", IOP_GROUP_CORRECT);
```
Comment (TurboGit) [inline src/iop/bilateral.cc]: the name was wrong, cut&paste... I have fixed that.

Comment (TurboGit) [reply src/iop/bilateral.cc]: And there was a missing include, seems like you didn't even test to compile the code!

Comment (PaoloAst) [reply src/iop/bilateral.cc]: Actually not, sorry. I'm not  experienced in DT and I didn't want to make something wrong

Comment (TurboGit) [reply src/iop/bilateral.cc]: You should have said so upfront I would have paid more attention to the code. Anyway all under control and working now.

Comment (PaoloAst) [reply src/iop/bilateral.cc]: You're definitely right. I will. I promise

**Pull request merged**